bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableConfiguration.java

```diff
@@ -178,6 +178,13 @@ public BigtableOptions toBigtableOptions() throws IOException {
    */
   public Configuration toHBaseConfig() {
     Configuration config = new Configuration();
+
```

Comment:
Ok, please note a note here then.
Also, the override should use value: BIGTABLE_ASYNC_MUTATOR_COUNT_DEFAULT
not BigtableOptions.BIGTABLE_DATA_CHANNEL_COUNT_DEFAULT